apps/prairielearn/src/ee/pages/studentCourseInstanceUpgrade/studentCourseInstanceUpgrade.html.ts

```diff
@@ -48,15 +48,15 @@ export function StudentCourseInstanceUpgrade({
                 ${PriceTable({ planNames: missingPlans, planPrices })}
 
                 <form method="POST">
-                  <div class="custom-control custom-checkbox mb-3">
+                  <div class="form-check mb-3">
```

Comment:
I don't have the env to check this page, so please double-check my work, though I don't expect an issue here.

Reply:
> Bootstrap 5 replaced .custom-control.custom-checkbox with .form-check. Please update your HTML.

I also didn't see any references to either class outside of the compat tool.